src/modules/utils/motordrivercontrol/MotorDriverControl.cpp

```diff
@@ -54,6 +63,21 @@ MotorDriverControl::~MotorDriverControl()
 // this will load all motor driver controls defined in config, called from main
 void MotorDriverControl::on_module_loaded()
 {
+    // Enabled?
+    if (!THEKERNEL->config->value(motor_driver_control_checksum, enable_checksum )->by_default(1)->as_bool()) {
```

Comment:
by_default needs to be false here, also not sure how that slipped in.

Reply:
It is actually superfluous not sure how it got in there as my original code did not have this at all. But it can't do any harm I suppose. I does make old configs not backward compatible though, so it should not have been added at all.

Reply:
should I remove these lines? The loop below this one checks anyway if a module is enabled or not.

Reply:
yes you can it serves no purpose.